main.js

```diff
@@ -4,8 +4,19 @@
 import './style.css';
 
 import { Earth } from './src/objects/Earth';
+import { clickedNothing } from './src/scene';
+import { MARKER_CLICK_OFF_CANVAS_DESELECTION } from './src/constants';
 
 const container = document.querySelector('[data-globe]');
 const planet = new Earth(container, {
   mapUrl: '/images/earthspec1k.jpg',
 });
+
+if (MARKER_CLICK_OFF_CANVAS_DESELECTION) {
+  window.document.addEventListener('click', (e) => {
+    const canvasElement = container.querySelector('canvas');
```

Comment:
I decided to keep all of this in `main.js` since I'm referencing `container` here and that isn't in `scene/index.js`